cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -459,14 +459,14 @@ func (t *cadenceValueMigrationReporter) Error(err error) {
 	}
 
 	if t.verboseErrorOutput {
-		t.log.Error().Msgf(
+		t.reportWriter.Write(fmt.Sprintf(
```

Comment:
Looking at the other uses of the report writers, AFAICS this should probably be written in a structured form, not just as a string, i.e. see e.g `reportWriter.Write(cadenceValueMigrationReportEntry{...})`.